pkg/manifests/fixtures/external_dns/full.json

```diff
@@ -30,7 +29,7 @@
         "creationTimestamp": null,
         "labels": {
           "app.kubernetes.io/managed-by": "aks-app-routing-operator",
-          "app.kubernetes.io/name": "external-dns-private"
```

Comment:
All were being given the private suffix due to a weird Golang reference thing. Not particularly worried about it because it will all be fixed in reconciliation if something rolls out.

Reply:
makes sense